src/components/Article/index.tsx

```diff
@@ -23,6 +23,7 @@ interface ArticleProps {
   /** The paragraphs of content for the story. */
   content: Content[]
   dropcap: boolean
+  style?: string
```

Comment:
We should add a JSDoc comment for this prop. See the footer and the docz docs (ha) for referenece https://www.docz.site/documentation/components-api#propstable

Reply:
got it boss

Reply:
woah no one's called me boss except 'scuse me boss man